src/core/Schemas.ts

```diff
@@ -269,6 +271,13 @@ export const MoveWarshipIntentSchema = BaseIntentSchema.extend({
   tile: z.number(),
 });
 
+export const QuickChatIntentSchema = BaseIntentSchema.extend({
+  type: z.literal("quick_chat"),
+  recipient: z.string(),
+  quickChatKey: z.string(),
```

Comment:
I'm worried people will inject arbitary text here. Can we convert this to a literal union:
```suggestion
  quickChatKey: z.union([z.literal("..."), ...)]),
```

Or to a table index:
```suggestion
  quickChatKey: z.number().nonNegative().max(..),
```